WEB/Src/HostingStartup/HostingStartup.Tests/HostingStartup.Tests.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.ApplicationInsights.HostingStartup.Tests</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.HostingStartup.Tests</AssemblyName>
+    <TargetFrameworks>net45;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
For hostingstartup, target only net45. This project is only consumed in .NET Framework environment.

Reply:
thanks for reviewing! i updated the file!